### PR TITLE
[#11553] Make account requests searchable in admin search page

### LINKED ISSUE #11553: Make account requests searchable in admin search page
- **Environment**:  live server at `V8.6.0`

**Description of feature/enhancement**

As suggested by @wkurniawan07, ~once #11396 is merged~, we should allow account requests to be searchable within the admin search page.

**Justification**

This will allow us to support resending links as well as deleting account requests.

**Existing similar features and their shortcomings**

~Once #11396 is merged~, There is currently no way to view existing account requests, in the admin search panel or otherwise. Coupled with the fact that there is a newly added restriction of unique email + institute combination, this creates confusion when admin needs to re-create new account (e.g. when the account activation email is not received).

### PR DESCRIPTION
**Pull request by samuelfangjw**
Fixes #11553

**Outline of solution**
Add account request section to admin search page.

**Screenshot of search page**

Screen Shot 2022-02-08 at 4 50 40 PM

### CODE REVIEW

File: src/e2e/resources/data/AdminSearchPageE2ETest.json [hunk 1]
```diff
@@ -132,5 +132,27 @@
     "isClosingEmailEnabled": true,
     "isPublishedEmailEnabled": true
     }
+  },
+  "accountRequests": {
+    "instructor1OfCourse1": {
+      "name": "Instructor1 of Course1",
+      "email": "ASearch.instructor1@gmail.tmt",
```
[the remark this answers is not in the document]
Comment (samuelfangjw) [reply src/e2e/resources/data/AdminSearchPageE2ETest.json:139]: I'll add the validation rule for `email`. Would that be enough?

[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/e2e/resources/data/AdminSearchPageE2ETest.json:139]: Yes, validating email is sufficient

**Comment by samuelfangjw** [comment]
Thanks for the review @jianhandev! I've made the changes :)

**Review by jianhandev — APPROVED** [review]
LGTM! 👍

**Review by wkurniawan07 — APPROVED** [review]
LGTM

**Pull request merged**